src/Domains/Domains.jl

```diff
@@ -125,14 +125,22 @@ end
 coordinate_type(::RectangleDomain{FT}) where {FT} =
     Geometry.Cartesian2DPoint{FT}
 
+abstract type AbstractSphere{FT} end
+struct EquiangularSphere{FT} <: AbstractSphere{FT} end
```

Comment:
isn't the sphere type more a property of the mesh than the domain?

Reply:
ex. an `EquiangularSphereMesh()` makes sense but a equiangular sphere domain does not really as the domain just defines the coordinates (LatLong) / coordinate extent (Radius).